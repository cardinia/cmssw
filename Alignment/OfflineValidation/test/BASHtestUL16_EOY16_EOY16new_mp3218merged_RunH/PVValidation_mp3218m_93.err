26-Sep-2019 12:51:30 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/762118CC-FA11-E911-8B0E-0CC47A4D766C.root
26-Sep-2019 12:51:34 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/762118CC-FA11-E911-8B0E-0CC47A4D766C.root
Begin processing the 1st record. Run 284025, Event 17655909, LumiSection 110 on stream 0 at 26-Sep-2019 12:51:42.631 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 12:57:39 CEST  Run: 284025 Event: 26073345
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 1001st record. Run 284025, Event 25807680, LumiSection 116 on stream 0 at 26-Sep-2019 12:57:50.524 CEST
Begin processing the 2001st record. Run 284025, Event 32644319, LumiSection 122 on stream 0 at 26-Sep-2019 13:03:42.006 CEST
Begin processing the 3001st record. Run 284025, Event 40863035, LumiSection 132 on stream 0 at 26-Sep-2019 13:09:20.609 CEST
Begin processing the 4001st record. Run 284025, Event 49114786, LumiSection 137 on stream 0 at 26-Sep-2019 13:15:19.921 CEST
26-Sep-2019 13:21:27 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/762118CC-FA11-E911-8B0E-0CC47A4D766C.root
26-Sep-2019 13:21:27 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/88AA4D9E-DA11-E911-A1FA-0CC47A7C35D8.root
26-Sep-2019 13:21:30 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/88AA4D9E-DA11-E911-A1FA-0CC47A7C35D8.root
Begin processing the 5001st record. Run 284025, Event 58249726, LumiSection 143 on stream 0 at 26-Sep-2019 13:21:36.802 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:24:40 CEST  Run: 284025 Event: 61243626
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 192
%MSG
Begin processing the 6001st record. Run 284025, Event 65577661, LumiSection 148 on stream 0 at 26-Sep-2019 13:27:49.145 CEST
Begin processing the 7001st record. Run 284025, Event 72709374, LumiSection 153 on stream 0 at 26-Sep-2019 13:33:51.288 CEST
Begin processing the 8001st record. Run 284039, Event 1300110, LumiSection 1 on stream 0 at 26-Sep-2019 13:38:30.754 CEST
Begin processing the 9001st record. Run 284039, Event 7974506, LumiSection 5 on stream 0 at 26-Sep-2019 13:38:35.195 CEST
Begin processing the 10001st record. Run 284039, Event 14983147, LumiSection 9 on stream 0 at 26-Sep-2019 13:38:39.239 CEST
Begin processing the 11001st record. Run 284039, Event 22755656, LumiSection 13 on stream 0 at 26-Sep-2019 13:38:42.573 CEST
Begin processing the 12001st record. Run 284039, Event 32045525, LumiSection 18 on stream 0 at 26-Sep-2019 13:38:47.103 CEST
Begin processing the 13001st record. Run 284039, Event 39444829, LumiSection 22 on stream 0 at 26-Sep-2019 13:38:50.958 CEST
Begin processing the 14001st record. Run 284039, Event 47077134, LumiSection 26 on stream 0 at 26-Sep-2019 13:38:54.378 CEST
Begin processing the 15001st record. Run 284044, Event 972958, LumiSection 1 on stream 0 at 26-Sep-2019 13:38:57.273 CEST
Begin processing the 16001st record. Run 284044, Event 10147666, LumiSection 7 on stream 0 at 26-Sep-2019 13:39:00.024 CEST
Begin processing the 17001st record. Run 284044, Event 16154742, LumiSection 12 on stream 0 at 26-Sep-2019 13:39:03.274 CEST
Begin processing the 18001st record. Run 284044, Event 23567381, LumiSection 17 on stream 0 at 26-Sep-2019 13:39:06.203 CEST
Begin processing the 19001st record. Run 284044, Event 33456251, LumiSection 23 on stream 0 at 26-Sep-2019 13:39:09.718 CEST
Begin processing the 20001st record. Run 284044, Event 39603744, LumiSection 28 on stream 0 at 26-Sep-2019 13:39:12.984 CEST
26-Sep-2019 13:39:13 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/88AA4D9E-DA11-E911-A1FA-0CC47A7C35D8.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       2        2
    2 fileAction           -s file_close                             2        2
    3 fileAction           -s file_open                              4        4

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          284025/26073345  284025/61243626  
    2 fileAction           PostProcessEvent PostGlobalEndRun 
    3 fileAction           pre-events       pre-events       PostProcessEvent

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 2                   2
System                  6                   6

dropped waiting message count 0
[8.957MB/8.957MB][100%][==================================================][8.957MB/s]  [8.957MB/8.957MB][100%][==================================================][8.957MB/s]  
