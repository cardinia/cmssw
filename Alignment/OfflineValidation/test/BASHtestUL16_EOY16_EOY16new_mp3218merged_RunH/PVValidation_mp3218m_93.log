000 (3189599.093.000) 09/26 11:25:39 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189599.093.000) 09/26 12:51:16 Started transferring input files
	Transferring to host: <128.142.101.161:9618?addrs=128.142.101.161-9618+[2001-1458-301-89--100-18d]-9618&noUDP&sock=3646_afcd_24509>
...
040 (3189599.093.000) 09/26 12:51:16 Finished transferring input files
...
001 (3189599.093.000) 09/26 12:51:16 Job executing on host: <128.142.101.161:9618?addrs=128.142.101.161-9618+[2001-1458-301-89--100-18d]-9618&noUDP&sock=1817_e3ac_3>
...
006 (3189599.093.000) 09/26 12:51:25 Image size of job updated: 150968
	91  -  MemoryUsage of job (MB)
	92172  -  ResidentSetSize of job (KB)
...
006 (3189599.093.000) 09/26 12:56:26 Image size of job updated: 577588
	559  -  MemoryUsage of job (MB)
	571964  -  ResidentSetSize of job (KB)
...
006 (3189599.093.000) 09/26 13:01:27 Image size of job updated: 578496
	560  -  MemoryUsage of job (MB)
	572872  -  ResidentSetSize of job (KB)
...
006 (3189599.093.000) 09/26 13:11:28 Image size of job updated: 579584
	561  -  MemoryUsage of job (MB)
	573952  -  ResidentSetSize of job (KB)
...
006 (3189599.093.000) 09/26 13:16:29 Image size of job updated: 579784
	561  -  MemoryUsage of job (MB)
	574152  -  ResidentSetSize of job (KB)
...
006 (3189599.093.000) 09/26 13:26:29 Image size of job updated: 603040
	584  -  MemoryUsage of job (MB)
	597380  -  ResidentSetSize of job (KB)
...
006 (3189599.093.000) 09/26 13:36:31 Image size of job updated: 605812
	587  -  MemoryUsage of job (MB)
	600312  -  ResidentSetSize of job (KB)
...
040 (3189599.093.000) 09/26 13:39:35 Started transferring output files
...
040 (3189599.093.000) 09/26 13:39:35 Finished transferring output files
...
005 (3189599.093.000) 09/26 13:39:35 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:46:08, Sys 0 00:00:08  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:46:08, Sys 0 00:00:08  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	6482  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	6482  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     9217        1   2091728 
	   Memory (MB)          :      587     2000      2000 
...
