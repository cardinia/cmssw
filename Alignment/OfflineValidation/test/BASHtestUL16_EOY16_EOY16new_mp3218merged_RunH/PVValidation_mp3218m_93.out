Job started at  Thu Sep 26 12:51:16 CEST 2019
batch dir: /pool/condor/dir_32637 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_284025_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 9.1M -rw-r--r--. 1 tkello zh 9.0M Sep 26 13:39 PVValidation_mp3218m_284025.root -rw-r--r--. 1 tkello zh 20K Sep 26 12:51 PVValidation_mp3218m_284025_cfg.py -rw-r--r--. 1 tkello zh 5.2K Sep 26 13:39 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 12:51 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 12:51 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:51 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:51 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:51 var
Job ended at  Thu Sep 26 13:39:35 CEST 2019
